components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js

```diff
@@ -995,7 +995,8 @@ define([
         },
 
         _labelSelectionOffset: function() {
-            var selectionOffset = this.isSelected() ? PedigreeEditorParameters.attributes.radius/1.4 : 0;
+            var labelsOffset = this.getHoverBox().getLabelsOffset();
+            var selectionOffset = this.isSelected() ? PedigreeEditorParameters.attributes.radius/1.4 + labelsOffset : 0;
```

Comment:
ideally getLabelsOffset() should incorporate this magical `PedigreeEditorParameters.attributes.radius/1.4` if we have the method now